src/embrakes/main.hpp

```diff
@@ -58,12 +58,10 @@ class Main : public Thread
     data::StateMachine     sm_data_;
     data::EmergencyBrakes  em_brakes_;
     data::Telemetry        tlm_data_;
-    int                    command_pins_[4];
-    int                    button_pins_[4];
-    StepperInterface*      brake_1;
-    // Stepper*               brake_2;
-    // Stepper*               brake_3;
-    // Stepper*               brake_4;
+    int                    command_pins_[2];
+    int                    button_pins_[2];
```

Comment:
A comment here might be helpful? I don't know a lot about sensors and it's not clear to me what this actually is. I'm assuming it's a pin index that can somehow be used to read/write a signal (similar to file descriptors). If this is the case then `int` might not be the best type? Afaik C++ does not guarantee the size of `int` so giving it an explicit size, i.e. using `uint8_t` ([which is what `Stepper` uses](https://github.com/Hyp-ed/hyped-2021/blob/d9e7aa6da275b26fca1ae641f03d74a06e5b7859/src/embrakes/stepper.hpp#L44)) may be better?

Reply:
Yeah, they're the GPIO pin numbers for sending commands and checking that the brakes activated. 

I definitely agree they should be commented, I found it quite difficult to work with the embrakes module due to lack of comments. I'll work on adding some comments in there.

Reply:
Changing to uint8_t seems like a good idea, I had left it like it was before as I wasn't sure if there was some reason for that.

Reply:
Needs to be uint16_t as uint8_t can't cover the possible GPIO pin values for BBB.